src/kvstore/kvstore_dist.h

```diff
@@ -32,11 +32,6 @@
 #include "mxnet/engine.h"
 #include "ps/ps.h"
 #include "./kvstore_dist_server.h"
-#if MKL_EXPERIMENTAL == 1
```

Comment:
Did you run distributed training test and verify it's fucntional?

Reply:
i haven't tried it. Where is the distributed training test?

Reply:
You could try running tests/nightly/dist_sync_kvstore.py

```cd tests/nightly && ../../tools/launch.py --launcher local -n 3 python dist_sync_kvstore.py``` to mock test locally

Reply:
Thank you for your instructions. I run the test on a GPU instance and it seems to pass the test. It seems the test is specifically written for sparse formats and gradient compression.

Currently, MKLDNN always uses the default layout for weight arrays during the training. I guess it may make more sense to train a model in a distributed environment?

Reply:
I tried `../../tools/launch.py --launcher local -n 3 python train_cifar10.py --kv-store dist_sync`. I let it run for a long time. It seems to converge well.